src/ngamsPlugIns/ngamsPlugIns/alma/ngamsAlmaMultipart.py

```diff
@@ -122,8 +122,14 @@ def ngamsAlmaMultipart(ngams_server, request_properties):
 
     file_id, final_filename, file_format = specific_treatment(staging_filename)
 
-    logger.debug("SDM multipart plug-in processing request for file with URI %s, file_format=%s, file_id=%s, file_version=%s," 
-                 " final_name=%s", request_properties.getFileUri(), file_format, file_id, file_version, final_filename)
+    #if request_properties.hasHttpPar("file_id"):
+    #    file_id = request_properties.getHttpPar("file_id")
+
+    #if request_properties.hasHttpPar("file_version"):
+    #    file_version = request_properties.getHttpPar("file_version")
+
```

Comment:
I'm curious about these commented code line that are being added -- are these being added on purpose, or were just left-overs from local experiments? If you'd rather keep them there for future reference that's fine, but in general I don't find it great to add commented code -- that's what scm is for :)

Reply:
I put them there as a reminder in case they could be useful. Perhaps it is better to check and use these arguments in the plugin. The plugins are very old so I am unsure what the correct behavior should be.